doc/content/gateways/troubleshooting/_index.md

```diff
@@ -211,3 +211,66 @@ The gateway location is stored in the Identity Server instead. The updated locat
 >
 >```bash
 > curl -H "Authorization: Bearer <api-key>" https://example.thethings.com/api/v3/gateways/<gateway-id>?field_mask=antennas
+
+## I do not see the downlink being scheduled by the Network Server. What do I do?
+
+If you notice your downlink is not being scheduled, check your gateway's Live data tab for the [`gs.down.send` event]({{< ref "/reference/api/events#event:gs.down.send" >}}). If you do not see this event, it means that the Network Server failed to schedule the downlink message to Gateway Server.
+
+You can simply try re-scheduling the downlink, and if the issue persists, check if the downlink is being properly scheduled from the Application Server to the Network Server.
+
+## When I schedule a downlink, I see the `gs.down.tx.fail` event in the Live data tab and the downlink message fails to be transmitted. Why?
+
+Seeing the [`gs.down.tx.fail` event]({{< ref "/reference/api/events#event:gs.down.tx.fail" >}}) in the gateway's Live data tab means the downlink message has been scheduled from the Gateway Server to the gateway, but the Gateway Server did not receive the ACK for that downlink. Some common causes and solutions for this issue:
+
+- High latency in the gateway backhaul - high latency usually occurs if the gateway and {{% tts %}} cluster are not geographically close, or the gateway is using a cellular or satellite backhaul. Always use {{% tts %}} cluster that is closest to your gateway's location, and make sure to check your gateway's Internet connection.
+- Many downlinks being scheduled at the same time via single gateway - this usually happens if a region is poorly equipped with gateways. The solution would be to deploy more gateways in the region, i.e. improve the network coverage.
+- Full duty cycles on the gateway - this can happen if the gateway is handling a lot of traffic, or the end devices are using SF10 or higher. Try adding more gateways to improve the network coverage in your region.
```

Comment:
Duty cycle enforcement is taken care of by TTS. If the duty cycle is reached,  TTS won't schedule downlinks and return a `duty cycle` error.